src/Point.h

```diff
@@ -19,26 +19,25 @@
 
 /**
  * @class Point Point.h "src/Point.h"
- * @brief Class to represent a 2D point in space.
+ * @brief Class to represent a 3D point in space.
  */
 class Point {
 
 private:
 
-  /** The Point's x-coordinate */
-  double _x;
-
-  /** The Point's y-coordinate */
-  double _y;
+  /** The Point's coordinates */
+  double* _xyz;
 
 public:
   Point();
   virtual ~Point();
-  void setCoords(const double x, const double y);
+  void setCoords(const double x, const double y, const double z=0.0);
```

Comment:
Ditto, I'm not sure it's necessary to make `z` the only parameter with a default value? Seems like none of them or all of them should have defaults unless I'm missing something.